redmine/issue.rb

```diff
@@ -35,17 +35,23 @@ def set_status(status)
     self
   end
 
-  def pull_request
+  def pull_requests
     field = @raw_data['issue']['custom_fields'].find { |f| f['id'] == 7 }
     return nil if field.nil?
     field['value']
   end
 
-  def set_pull_request(url)
+  def set_pull_requests(url)
     @raw_data['issue']['custom_field_values'] = {'7' => url}
     self
   end
 
+  def add_pull_request(url)
+    current_pull_requests = pull_requests
+    current_pull_requests << url
```

Comment:
Sorry, I mean de-duplicate the URLs in the list.  If this code runs again (when a PR's updated) then it might add the PR URL twice to the same ticket.  We might only want to update if the existing issue.pull_requests list doesn't contain the URL, or always update but call #uniq?